bootloaders/riotboot/README.md

```diff
@@ -94,17 +100,9 @@ Dedicated make targets are available to build and flash several slots:
 
 In particular, if one wants to be sure to boot a particular image, using the
 target `riotboot/flash-extended-slot0` is the way to go (resulting in only
-slot 0 being valid, thus being booted).
-
-## Flashing examples
-
-The following sequence of commands tests building, flashing and booting slot 0,
-then slot 1. tests/riotboot prints out the current booted slot in the shell.
-
-To test building, flashing and booting the first slot:
-
-`BOARD=samr21-xpro APP_VER=$(date +%s) make -C tests/riotboot/ riotboot/flash-combined-slot0 test`
+slot 0 being valid, thus being booted). This is done automatically by `make
+flash` if the `riotboot` feature is used.
 
-For the second slot:
+## Testing riotboot
 
-`BOARD=samr21-xpro APP_VER=$(date +%s) make -C tests/riotboot/ riotboot/flash-slot1 test`
+See tests/riotboot/README.md.
```

Comment:
Can you put this as a link? [tests/riotboot/README.md](https://github.com/RIOT-OS/RIOT/blob/master/tests/riotboot/README.md)

Reply:
good idea, fixed